src/printer.js

```diff
@@ -2724,6 +2656,7 @@ function printStatementSequence(path, options, print) {
 
     // Just in case the AST has been modified to contain falsy
     // "statements," it's safer simply to skip them.
+    /* istanbul ignore if */
     if (!stmt) {
       return;
     }
```

Comment:
Yeah I might do that is a follow up.

I finished reviewing printer.js, but still need to look at the rest of the codebase.